### DISCUSSION TITLE
DGB-Odocrypt support??

### DISCUSSION THREAD

**Discussion by seperatis** [coinfoundry/miningcore#849] [Ideas]
Does Miningcore currently support DGB Odocrypt?

**Comment by seperatis** [comment]
bump

**Comment by oliverw** [comment]
Won't happen unless someone submits a PR. I've experimented with Odo before they actually introduced it, but failed due to lack of a proper hardware miner.

**Comment by calvintam236** [comment]
@oliverw I am reopening this just for visibility for PR contribution.

**Reply by seperatis** [reply]
Odocrypt is no longer viable.....

 from what I understand the DGB devs are working with the hardware manufacturers at Hash Alt Coin to make Odocrypt v2 aka Odocrypt+

This decision was made after finding out there are Odocrypt ASICs secretly mining on the network.

So any Odocrypt implementation is wasting time until Odocrypt v2/Odocrypt+ is released for DGB.

**Comment by oliverw** [comment]
Interesting. Sneaky chinese 😁
